### ISSUE TITLE
[Request] Sponsorblock support 

### ISSUE THREAD

**Issue by sTiKyt** [TeamNewPipe/NewPipe#8274]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

I'd like to request reconsidering your opinion on Sponsor Block

### Why do you want this feature?

I believe sponsorblock support is essential for most people coming here after Vanced was closed

I've seen [#2554](https://github.com/TeamNewPipe/NewPipe/issues/2554#issuecomment-732428678) but response makes no logical sense, people are simply asking for it to be optional, not obligatory, it has multiple switches you can turn to your preferences and it's not for Ads exclusively, it also helps you skip Intermissions, Intro Animations, Endcards, Credits, Reminders to interact (like and sub), Self Promotions, Non-music sections in music videos, recaps and fillers.

As you can see it has significant value for people trying to keep their watchtime efficient

### Additional information

There is no practical reason to not include it, there is even [fork with this functionality already](https://github.com/polymorphicshade/NewPipe#newpipe-x-sponsorblock)

Sorry for my ignorance if there are critical issues behind your decision, and sorry for duplicate, but [#2554](https://github.com/TeamNewPipe/NewPipe/issues/2554#issuecomment-732428678) had comments closed

**Comment by SameenAhnaf** [comment]
>most people coming here after Vanced was closed

Newpipe never intended to be an alternative for Vanced. See: https://github.com/TeamNewPipe/NewPipe/issues/8025

>people are simply asking for it to be optional

Even if Sponsorblock had no flaw, it should have been optional. But that's not the key problem.

>As you can see it has significant value for people trying to keep their watchtime efficient

TeamNewpipe have made their [position](https://newpipe.schabi.org/blog/pinned/newpipe-and-online-advertising/) clear.

>there is even [fork with this functionality already](https://github.com/polymorphicshade/NewPipe#newpipe-x-sponsorblock)

The fork creator whole-heartedly supports Newpipe's approach not to include Sponsorblock and is willing to maintain fork. See: https://github.com/TeamNewPipe/NewPipe/pull/3205#issuecomment-733264819

So, what's the big deal? The code has to be regularly maintained anyway.

**Comment by litetex** [comment]
https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed+Sponsorblock

**Comment by qol3m** [comment]
> > there is even [fork with this functionality already](https://github.com/polymorphicshade/NewPipe#newpipe-x-sponsorblock)
> 
> The fork creator whole-heartedly supports Newpipe's approach not to include Sponsorblock and is willing to maintain fork. See: [#3205 (comment)](https://github.com/TeamNewPipe/NewPipe/pull/3205#issuecomment-733264819)
> 
> So, what's the big deal? The code has to be regularly maintained anyway.

I can't get a different fork for every feature I need. I need all in one app. 

> TeamNewpipe have made their [position](https://newpipe.schabi.org/blog/pinned/newpipe-and-online-advertising/) clear.

Their position is flawed in two ways:

People who don't want to watch ads will never watch ads. There is nothing on earth that Linus could possibly recommend me that I would buy. Literally not a single thing on earth. I. do. not. care.

They also mention "recurring donations" as a method of supporting creators, and don't want to "gatekeep", yet there is no feature in newpipe that lets me watch with my youtube premium account. Even if there was, they fail to realize that paying for premium and still having hardcoded sponsors isn't an option for many.

I'll support the creators. You just have to let me. With ads of any kind it's a hard pass. They bring in a tiny fraction of money that my time spent watching the ad is worth.

**Comment by sTiKyt** [comment]
> > > there is even [fork with this functionality already](https://github.com/polymorphicshade/NewPipe#newpipe-x-sponsorblock)
> > 
> > 
> > The fork creator whole-heartedly supports Newpipe's approach not to include Sponsorblock and is willing to maintain fork. See: [#3205 (comment)](https://github.com/TeamNewPipe/NewPipe/pull/3205#issuecomment-733264819)
> > So, what's the big deal? The code has to be regularly maintained anyway.
> 
> I can't get a different fork for every feature I need. I need all in one app.
> 
> > TeamNewpipe have made their [position](https://newpipe.schabi.org/blog/pinned/newpipe-and-online-advertising/) clear.
> 
> Their position is flawed in two ways:
> 
> People who don't want to watch ads will never watch ads. There is nothing on earth that Linus could possibly recommend me that I would buy. Literally not a single thing on earth. I. do. not. care.
> 
> They also mention "recurring donations" as a method of supporting creators, and don't want to "gatekeep", yet there is no feature in newpipe that lets me watch with my youtube premium account. Even if there was, they fail to realize that paying for premium and still having hardcoded sponsors isn't an option for many.
> 
> I'll support the creators. You just have to let me. With ads of any kind it's a hard pass. They bring in a tiny fraction of money that my time spent watching the ad is worth.

This tbh

**Comment by allanlaal** [comment]
TLDR/googlers: Use this fork that has SponsorBlock + Return Dislikes:
https://github.com/polymorphicshade/NewPipe
